Textile/Textile.sublime-syntax

```diff
@@ -1,225 +1,1218 @@
+        - include: inlines
+        - include: html
+    - match: |-
+        (?x:
+          ^
+          (h4)
+          ({{indentation_optional}})
+          ({{alignment_optional}})
+          ({{attributes_optional_class_style}})
+          (\.)
+          [ ](?=\S)
+        )
+      captures:
+        1: entity.name.tag.heading.textile
+        2: constant.other.indentation.textile
+        3: constant.other.alignment.textile
+        4: entity.name.type.textile
+        5: entity.name.tag.heading.textile
+      push:
+        - meta_scope: markup.heading.4.textile
+        - meta_content_scope: meta.toc-list.textile
+        # before-eol-pop
+        - match: '$'
+          pop: 1
+        - include: inlines
+        - include: html
+    - match: |-
+        (?x:
+          ^
+          (h5)
+          ({{indentation_optional}})
+          ({{alignment_optional}})
+          ({{attributes_optional_class_style}})
+          (\.)
+          [ ](?=\S)
+        )
+      captures:
+        1: entity.name.tag.heading.textile
+        2: constant.other.indentation.textile
+        3: constant.other.alignment.textile
+        4: entity.name.type.textile
+        5: entity.name.tag.heading.textile
+      push:
+        - meta_scope: markup.heading.5.textile
+        - meta_content_scope: meta.toc-list.textile
+        # before-eol-pop
+        - match: '$'
+          pop: 1
+        - include: inlines
+        - include: html
+    - match: |-
+        (?x:
+          ^
+          (h6)
+          ({{indentation_optional}})
+          ({{alignment_optional}})
+          ({{attributes_optional_class_style}})
+          (\.)
+          [ ](?=\S)
+        )
+      captures:
+        1: entity.name.tag.heading.textile
+        2: constant.other.indentation.textile
+        3: constant.other.alignment.textile
+        4: entity.name.type.textile
+        5: entity.name.tag.heading.textile
+      push:
+        - meta_scope: markup.heading.6.textile
+        - meta_content_scope: meta.toc-list.textile
+        # before-eol-pop
+        - match: '$'
+          pop: 1
+        - include: inlines
+        - include: html
+
+###[ BLOCKS / PRE-FORMATTED TEXT ]#############################################
+
+  # https://textile-lang.com/doc/pre-formatted-text
+
+  pre-blocks:
+    - include: extended-blocks-pre
+    - include: blocks-pre
+
+  extended-blocks-pre:
+    - match: |-
+        (?x:
+          ^
+          (pre)
+          ({{indentation_optional}})
+          ({{alignment_optional}})
+          ({{attributes_optional_class_style}})
+          (\.\.)
+          [ ](?=\S)
+        )
+      captures:
+        1: entity.name.tag.pre.textile
+        2: constant.other.indentation.textile
+        3: constant.other.alignment.textile
+        4: entity.name.type.textile
+        5: entity.name.tag.pre.textile
+      push: extended-block-pre-body
+
+  extended-block-pre-body:
+    - meta_scope: markup.raw.textile
+
+    - include: next-block-pop
```

Comment:
Do we really need black lines between those rules? 

1. They are pretty clearly distinguished via syntax highlighting (PackageDev).
2. I'd in general not seperate each pattern by black lines as it is just too much, especially in such simple contexts it looks rather off.


```suggestion
    - meta_scope: markup.raw.textile
    - include: next-block-pop
```